Pyrado/pyrado/exploration/stochastic_action.py

```diff
@@ -395,14 +395,14 @@ def __init__(self, policy: Policy, eps: float = 1.0, eps_schedule_gamma: float =
 
     def eval(self):
         """ Call PyTorch's eval function and set the deny every exploration. """
-        super(Policy, self).eval()
+        super().eval()
```

Comment:
good to adapt to the new style

Reply:
Especially as Police was not the correct super class ;)